src/workloads/workloads.csproj

```diff
@@ -0,0 +1,192 @@
+
+    <!-- Project generated from the workload manfiests to download all the workload pack packages -->
+    <GatherPacksProject>$(WorkloadIntermediateOutputPath)src\gather\GatherPacks.csproj</GatherPacksProject>
+  </PropertyGroup>
+
+  <!-- Arcade -->
+  <PropertyGroup>
+      <!-- Temp directory for light command layouts -->
+      <LightCommandObjDir>$(ArtifactsObjDir)/LightCommandPackages</LightCommandObjDir>
+      <!-- Directory for the zipped up light command package -->
+      <LightCommandPackagesDir>$(ArtifactsNonShippingPackagesDir)</LightCommandPackagesDir>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <WixToolsetPath>$(PkgWix)\tools</WixToolsetPath>
+    <SwixPluginPath>$(PkgMicroBuild_Plugins_SwixBuild)</SwixPluginPath>
+    <SwixBuildTargets>$(SwixPluginPath)\build\MicroBuild.Plugins.SwixBuild.targets</SwixBuildTargets>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Add additional workload manifest packages here -->
+    <ManifestPackages Include="Microsoft.NET.Workload.Mono.ToolChain.Manifest-6.0.100" Version="$(MonoWorkloadManifestVersion)" GeneratePathProperty="true" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Workloads" Version="$(MicrosoftDotNetBuildTasksWorkloadsPackageVersion)" GeneratePathProperty="true" />
+    <PackageReference Include="WiX" Version="$(WixPackageVersion)" GeneratePathProperty="true" />
+    <PackageReference Include="MicroBuild.Plugins.SwixBuild" Version="$(SwixPackageVersion)" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Installers" Version="$(MicrosoftDotNetBuildTasksInstallersPackageVersion)" GeneratePathProperty="True" />
+    <PackageReference Include="@(ManifestPackages)" />
+  </ItemGroup>
+
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
+
+  <Target Name="GatherWorkloadPacks">
+    <ItemGroup>
+      <ManifestFile Include="@(ManifestPackages->'$(NuGetPackageRoot)%(Identity)\%(Version)\data\WorkloadManifest.json')" />
+    </ItemGroup>
+
+    <GetWorkloadPackPackageReferences
+      PackageSource="$(PackageSource)"
+      ExcludedPackIds="Microsoft.NETCore.App.Runtime.AOT.win-x86.Cross.browser-wasm"
+      ProjectFile="$(GatherPacksProject)"
+      ManifestFiles = "@(ManifestFile)" />
+
+    <!-- All the workload pack packages are copied to a single source location. They will get unzipped and the tasks allow for 
+         deleting arbitrary files from them, so we cannot use the installed copies in the NuGet cache. -->
+    <MSBuild Projects="$(GatherPacksProject)" Targets="restore;build" />
+  </Target>
+
+  <Target Name="Build" DependsOnTargets="GatherWorkloadPacks;GenerateVersions">
+    <ItemGroup>
+      <!-- Overrides for Visual Studio setup generation. If the workload definition IDs change,
+           these must be updated. -->
+      <ComponentResources Include="microsoft-net-sdk-blazorwebassembly-aot" Title=".NET WebAssembly Build Tools"
```

Comment:
@danroth27 @lewing can you take a look at the title/description. This is what would show in VS setup UI

Reply:
Looks good to me.